packages/core/src/routes/custom-phrase.ts

```diff
@@ -34,7 +34,7 @@ export default function customPhraseRoutes<T extends AuthedRouter>(
     '/custom-phrases',
     koaGuard({
       response: CustomPhrases.guard.array(),
-      status: [200],
+      status: [200, 404],
```

Comment:
It would be empty array if there's nothing to return, not likely a 404 error, IMO